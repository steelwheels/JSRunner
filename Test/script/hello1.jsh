
for(var i=0 ; i<3 ; i++){
	> echo "hello, world ${i}" | tr [a-z] [A-Z] ; echo "Good morning"
}
