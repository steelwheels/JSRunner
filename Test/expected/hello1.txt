HELLO, WORLD 0
Good morning
HELLO, WORLD 1
Good morning
HELLO, WORLD 2
Good morning
